packages/deployment/scripts/install-deps.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+set -e
+
+# Install Terraform.
+TERRAFORM_VERSION=0.11.14
+
+UNAME_S=$(uname -s | tr A-Z a-z)
+
+case $UNAME_S in
+*) TERRAFORM_OS=$UNAME_S ;;
+esac
+
+UNAME_M=$(uname -m)
+case $UNAME_M in
+x86_64) TERRAFORM_ARCH=amd64 ;;
+*) TERRAFORM_ARCH=$UNAME_M ;;
+esac
+
+TERRAFORM_RELEASE=terraform_${TERRAFORM_VERSION}_${TERRAFORM_OS}_${TERRAFORM_ARCH}
+curl https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_RELEASE}.zip > terraform.zip && \
```

Comment:
shellcheck would tell you to quote this so that it means the same thing if a field separator snuck into one of the variables.

Reply:
I installed the Shellcheck VSCode plugin.  Very nice!